serialization/microsoft_cfb.ksy

```diff
@@ -20,25 +20,25 @@ doc: |
       directory entry (subsequent sectors should be loaded according to FAT).
 seq:
   - id: header
-    type: cfb
+    type: cfb_header
```

Comment:
why not just `header`?

Reply:
Originally it was `cfb_header`, then I changed it to `cfb`, and then reverted it back but probably `header` should be enough.